client/src/sockets/helpers/index.js

```diff
@@ -12,6 +12,20 @@ export const listEventsController = (event, data, dispatch) => {
         `/${isCohortMember ? `cohort/${cohortId}` : 'dashboard'}`
       );
     }
+    case ListEvents.REMOVE_MEMBER_SUCCESS: {
+      const { cohortId, isCohortMember } = data;
+
+      if (cohortId) {
+        dispatch({ type: event, payload: data });
+        history.replace(
+          `/${isCohortMember ? `cohort/${cohortId}` : 'dashboard'}`
+        );
+
+        return window.location.reload();
```

Comment:
why `history.replace()` and `widnow.location.reload()` together? Maybe try `window.location = 'path'` instead of using together `hisotry.replace()` and `window.reload()` or check solution:
```
case ListEvents.LEAVE_ON_TYPE_CHANGE_SUCCESS: {
      const { cohortId, isCohortMember, listId } = data;

       dispatch({ type: ListEvents.DELETE_SUCCESS, payload: listId });

       return history.replace(
        `/${isCohortMember ? `cohort/${cohortId}` : 'dashboard'}`
      );
    }
```